src/scala/microsoft-spark-2.4.x/src/main/scala/org/apache/spark/api/dotnet/SerDe.scala

```diff
@@ -138,6 +139,11 @@ object SerDe {
     (0 until len).map(_ => readRow(in)).toList.asJava
   }
 
+  def readAnyArr(in: DataInputStream): Seq[Any] = {
```

Comment:
nit: I think this should be `readObjectArr`

Reply:
Done, thanks!